Makefile

```diff
@@ -63,7 +63,7 @@ dist/eventsource-controller-%:
 
 eventsource-controller-image: dist/eventsource-controller-linux-amd64
 	DOCKER_BUILDKIT=1 docker build -t $(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG)  --target eventsource-controller -f $(DOCKERFILE) .
-	@if [ "$(DOCKER_PUSH)" = "true" ] ; then  docker push $(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG) ; fi
+	@if [ "$(DOCKER_PUSH)" = "true" ] ; then  docker push $(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG) ; docker tag $(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG) quay.io/$(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG) ; docker push quay.io/$(IMAGE_PREFIX)eventsource-controller:$(IMAGE_TAG)  ; fi
```

Comment:
I would prefer not to retag the image here but let the github workflow build one more time, the change here will break the ppl who use `make` command to build their own images.

Reply:
Great point. I will rework